hrjobcontract/js/src/job-contract/controllers/modal/modal-change-reason.js

```diff
@@ -1,11 +1,12 @@
 define([
     'common/moment',
-    'job-contract/controllers/controllers'
+    'job-contract/controllers/controllers',
+    'job-contract/services/contract'
 ], function (moment, controllers) {
     'use strict';
 
-    controllers.controller('ModalChangeReasonCtrl',['$scope','$uibModalInstance', 'content', 'date', 'reasonId', '$log',
-        function ($scope, $modalInstance, content, date, reasonId, $log) {
+    controllers.controller('ModalChangeReasonCtrl',['$scope','$uibModalInstance', 'content', 'date', 'reasonId', '$log', 'settings', 'ContractRevisionService',
+        function ($scope, $modalInstance, content, date, reasonId, $log, settings, ContractRevisionService) {
```

Comment:
group similar together like $log must be along with other system dependencies like $scope